bin/testEnvLocalK8S.sh

```diff
@@ -15,19 +15,19 @@ export PATH=${GOPATH}/bin:${PATH}
 export OUT=${TOP}/out
 
 # components used in the test (starting with circleci for consistency, eventually ci will use this)
-export K8S_VER=v1.7.4
+export K8S_VER=${K8S_VER:-v1.9.2}
+export ETCD_VER=${ETCD_VER:-v3.2.15}
+
 export MASTER_IP=127.0.0.1
 export MASTER_CLUSTER_IP=10.99.0.1
-export ETCD_VER=v3.2.15
 
 # TODO: customize the ports and generate a local config
 export KUBECONFIG=${TOP}/src/istio.io/istio/.circleci/config
 
-export USE_BAZEL=0
-
-${ISTIO_GO}/bin/init.sh
+#${ISTIO_GO}/bin/init.sh
```

Comment:
why comment it out? We need envoy locally, and need to run the dep ensure thing right?

Reply:
There is a strange log - showing some download going on, when we run the tests.
I was trying to pinpoint it. 

During integration test execution we should already have all the binaries and download done by the previous stages ( build ).

Yet it doesn't seem to be it.

Reply:
It's be nice to get rid of all the direct callers of init.sh in favor of calling via make (e.g., "make init", or another target).  I didn't want to duplicate Makefile's logic for determining the go/out/... directory in init.sh, so if init.sh is called directly it just inits in go/bin.  So, if a script does run "init.sh" and then invokes a make target then you'll see it init proxy (if not more) twice.